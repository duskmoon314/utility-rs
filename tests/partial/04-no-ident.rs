use utility_types::Partial;

#[derive(Partial)]
#[partial(ident)]
pub struct A {
    a: usize,
    b: Option<usize>,
}

fn main() {}
