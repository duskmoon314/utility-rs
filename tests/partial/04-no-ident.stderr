error: Unexpected meta-item format `word`
 --> tests/partial/04-no-ident.rs:4:11
  |
4 | #[partial(ident)]
  |           ^^^^^
